ideascube/tests/test_models.py

```diff
@@ -72,10 +72,19 @@ def test_user_data_fields_should_return_labels_and_values(settings):
 class JSONModel(models.Model):
     class Meta:
         app_label = 'ideascube'
+        managed = False
 
     data = JSONField()
 
 
+@pytest.fixture()
+def json_table():
+    from django.db import connections
+    cursor = connections['default'].cursor()
+    cursor.execute("CREATE TABLE ideascube_jsonmodel "
+                   "(id, data)")
```

Comment:
That a good question.
I wonder if in fact, it just do not create the table as there is no migration that create the model.
So it fail to use it. If we run test without migration, django create all table from the defined model (and doesn't care of the migration script) and tests pass.